package.json

```diff
@@ -24,7 +24,13 @@
         "lint:ts": "cross-env tsc --noEmit && cross-env eslint './src/**/*.{ts,tsx,js,jsx}' --fix",
         "docs": "cross-env typedoc && node ./website/docs/scripts/fixdocs.js",
         "qode": "cross-env node ./scripts/qode.js",
-        "prepublishOnly": "cross-env npm run build"
+        "prepublishOnly": "cross-env npm run build",
+        "example-modelview_1_readonly": "node ./scripts/qode.js dist/examples/modelview_1_readonly.js",
+        "example-modelview_2_formatting": "node ./scripts/qode.js dist/examples/modelview_2_formatting.js",
+        "example-modelview_3_changingmodel": "node ./scripts/qode.js dist/examples/modelview_3_changingmodel.js",
+        "example-modelview_4_headers": "node ./scripts/qode.js dist/examples/modelview_4_headers.js",
+        "example-modelview_5_edit": "node ./scripts/qode.js dist/examples/modelview_5_edit.js",
+        "example-modelview_buddy": "node ./scripts/qode.js dist/examples/modelview_buddy.js"
```

Comment:
do we really need to add these here ?
We could just mention this in the readme right ?
something like 

`yarn qode  dist/examples/modelview_5_edit.js`

Reply:
:+1: